cli/cmd/generate_dockerfile_test.go

```diff
@@ -0,0 +1,111 @@
+package cmd_test
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/codesphere-cloud/cs-go/api"
+	"github.com/codesphere-cloud/cs-go/api/openapi_client"
+	"github.com/codesphere-cloud/cs-go/cli/cmd"
+	"github.com/codesphere-cloud/cs-go/pkg/cs"
+	"github.com/codesphere-cloud/cs-go/pkg/export"
+	"github.com/codesphere-cloud/cs-go/pkg/git"
+)
+
+var _ = Describe("GenerateDockerfile", func() {
+	var (
+		memoryFs     *cs.FileSystem
+		mockClient   *cmd.MockClient
+		mockEnv      *cmd.MockEnv
+		mockExporter *export.MockExporter
+		mockGit      *git.MockGit
+		c            *cmd.GenerateDockerfileCmd
+		wsId         int
+		repoStr      string
+		ws           api.Workspace
+	)
+
+	BeforeEach(func() {
+		mockClient = cmd.NewMockClient(GinkgoT())
+		memoryFs = cs.NewMemFileSystem()
+		mockEnv = cmd.NewMockEnv(GinkgoT())
+		mockExporter = export.NewMockExporter(GinkgoT())
+		mockGit = git.NewMockGit(GinkgoT())
+
+		defaultInput := "ci.yml"
+		defaultOutput := "./export"
+		wsId = 1
+		repoStr = "https://fake-git.com/my/repo.git"
+		c = &cmd.GenerateDockerfileCmd{
+			Opts: cmd.GenerateDockerfileOpts{
+				GlobalOptions: cmd.GlobalOptions{
+					Env:         mockEnv,
+					WorkspaceId: &wsId,
+				},
+				Input:  &defaultInput,
+				Output: &defaultOutput,
+			},
+		}
+	})
+
+	Context("the baseimage is not provided", func() {
+		FIt("should return an error", func() {
```

Comment:
```suggestion
		It("should return an error", func() {
```
FIt focusses the suite only on tests with F, so it's more something to do locally, but shouldn't be checked in.